packages/vue/src/pages/crm-contact-list/crm-contact-list.vue

```diff
@@ -77,7 +80,7 @@
           location="after"
           locate-in-menu="auto"
         >
-          <div class="separator" />
+          <div><div class="separator" /></div>
```

Comment:
Could you move it to a new line? Like:
<div>
  <div class="separator" />
</div>